pegasus/sites.v3/code.org/public/project-ideas.md.partial

```diff
@@ -0,0 +1,131 @@
+<h2><a href="https://code.org/educate/spritelab"><strong>Sprite Lab</strong></a></h2>
+<div class="col-33" style="padding-right: 20px;">
+    <img src="/images/spritelab/rikki.jpg">
+</div>
+<div class="col-66">
+    <p>Sprite Lab is a block-based programming environment where you can make simple animations and games with objects and characters that interact with each other. Create an animation or game then share your app!
+</div>
+<div style="clear: both; margin-bottom: 20px"></div>
+<table>
+    <thead>
+        <tr>
+            <th>Sprite Lab Project Ideas</th>
+            <th></th>
+            <th></th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h3><strong>Virtual Pet</strong></h3><p>Drag the food or the toy to the pet and watch the sprites interact.</p><img src="/images/spritelab/pet.png" style="max-width:50%"/><br><br><p><a href="https://studio.code.org/s/coursef-2021/lessons/4/levels/1">Try it out!</a></p></td>
+            <td><h3><strong>Greeting Card</strong></h3><p>Enter your name and age to get a personalized greeting.</p><img src="/images/spritelab/birthday.png" style="max-width:50%"/><br><br><p><a href="https://studio.code.org/s/coursef-2021/lessons/8/levels/2/sublevel/1">Try it out!</a></p></td>
+            <td><h3><strong>Sky Dancer</strong></h3><p>Collect the coins. Avoid the clouds. The sky's the limit!</p><img src="/images/spritelab/sky.png" style="max-width:50%"/><br><br><p><a href="https://studio.code.org/s/coursef-2021/lessons/20/levels/1/sublevel/2">Try it out!</a></p></td>
+        </tr>
+    </tbody>
+</table>
+
+# Project Ideas for Middle and High School Students
+<p> <a href="https://code.org/educate/weblab">Web Lab<a>, <a href="https://code.org/educate/gamelab">Game Lab</a> and <a href="https://code.org/educate/applab">App Lab</a> can be used by anyone but are targeted at middle and high school students. Programming in a brand new environment can be challenging and intimidating. We recommend giving yourself sufficient time to learn to use them. Don't give up! Below you can find an overview of each tool and also how to practice using it. For more information about full course offerings, check out <a href="https://code.org/educate/csd">CS Discoveries</a> for middle school or <a href="https://code.org/educate/csp">CS Principles</a> for high school.</p>
+<h2><a href="https://code.org/educate/weblab"><strong>Web Lab</strong></a></h2>
+<div class="col-33" style="padding-right: 20px;">
+    <img src="/images/athome/web-lab.jpg">
+    <p><i>Photo courtesy of Fundación Sadosky</i><p>
+</div>
+<div class="col-66">
+    <p>Web Lab is a programming environment where students can make simple web pages using HTML and CSS. Design your web pages and share your site in seconds. Recommended for students who have completed at least Chapter 1 of <a href="https://curriculum.code.org/csd/unit2/">Web Development</a>.</p>
```

Comment:
Suggestion for updating link to something evergreen.
Before: 
`Chapter 1 of <a href="https://curriculum.code.org/csd/unit2/">Web Development</a>`

After:
`Chapter 1 of Web Development from <a href="https://studio.code.org/courses/csd">CS Discoveries</a>`

@dancodedotorg - thoughts?

Reply:
Yes with a but: LP/EPT has been talking about getting rid of some redirects (like /courses/csd redirecting to the current course: /courses/csd-2021) We should confirm this redirect will remain as LP/EPT looks at getting rid of other redirects. If not, we should use /courses/csd-2021 and make a note to update this page every year.